src/Tizen.NUI/src/internal/Application.cs

```diff
@@ -1213,11 +1222,22 @@ public static Application New(int argc, string stylesheet, Application.WindowMod
 
         public static Application New(string[] args, string stylesheet, Application.WindowMode windowMode, Rectangle positionSize)
         {
-            int argc = args.Length;
-            string argvStr = string.Join(" ", args);
+            Application ret = null;
+            try
+            {
+                int argc = args.Length;
+                string argvStr = string.Join(" ", args);
+
+                ret = new Application(NDalicPINVOKE.Application_New_WithWindowSizePosition(argc, argvStr, stylesheet, (int)windowMode, Rectangle.getCPtr(positionSize)), true);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            }
+            catch (Exception exception)
+            {
+                Tizen.Log.Fatal("NUI", "message : " + exception.Message);
+                Tizen.Log.Fatal("NUI", "Stack Trace : " + exception.StackTrace);
```

Comment:
as far as I know, if exception occurs, BT(back trace) call stack is shown by DOTNET_LAUNCHER. isn't it same? I don't know well, so I am just asking.

Reply:
Fixed